libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -464,6 +472,30 @@ bool AP_BattMonitor::arming_checks(size_t buflen, char *buffer) const
     return true;
 }
 
+// Check's each smart battery instance for its powering off state and broadcasts notifications
+void AP_BattMonitor::checkPoweringOff(void)
+{
+    static bool powerOffNotified[AP_BATT_MONITOR_MAX_INSTANCES];
```

Comment:
Presumably a power off here means the entire vehicle is shutting down, why do you need this per instance?